decidim-conferences/config/locales/en.yml

```
@@ -352,7 +352,7 @@ en:
             invites: Invites
             search: Search
           new:
-            explanation: The participant will be invited to join a conference. If her email is not registered she will be invited to the organization as well.
+            explanation: The participant will be invited to join a conference. If their email is not registered, it'll be invited to the organization as well.
```

It should be "they'll" here, I think.
```suggestion
            explanation: The participant will be invited to join a conference. If their email is not registered, they'll be invited to the organization as well.
```